core/query/annotated.go

```diff
@@ -34,7 +34,7 @@ type AnnotatedInput struct {
 	Amount          uint64             `json:"amount"`
 	IssuanceProgram chainjson.HexBytes `json:"issuance_program,omitempty"`
 	ControlProgram  chainjson.HexBytes `json:"control_program,omitempty"`
-	SpentOutput     *SpentOutput       `json:"spent_output,omitempty"`
```

Comment:
I'd suggest leaving `spent_output` as is for compatibility, but add `spent_output_id` as a separate item because we'll remove or replace `spent_output` pair anyway after txgraph redesign.